Content.Server/Implants/SubdermalImplantSystem.cs

```diff
@@ -98,6 +101,10 @@ private void OnScramImplant(EntityUid uid, SubdermalImplantComponent component,
         if (!TryComp<ScramImplantComponent>(uid, out var implant))
             return;
 
+        // If we do not stop pulling then if the user is cuffed they will teleport they will be teleporter with their captor.
```

Comment:
English fail

Reply:
Shush I'm running on like 3-4 hours of sleep. I'll fix it one second.